projects/core/src/lib/shared/helpers/attachment.helper.ts

```diff
@@ -0,0 +1,32 @@
+import { AttachmentAttributeModel, AttachmentMetadataModel } from '@tailormap-viewer/api';
+import { formatDate } from '@angular/common';
+import { inject, Injectable, LOCALE_ID } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AttachmentHelper {
```

Comment:
To avoid confusion I would suggest to rename this to AttachmentService and also put this in a services folder. Currently we dont have helpers using injected services.